__tests__/app.test.js

```diff
@@ -71,21 +71,36 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/1")
       .expect(200)
       .then(({ body }) => {
-        expect(body.length).toBeGreaterThan(0);
-        expect(body).toBeSortedBy("date", { descending: true });
-        expect(body).toMatchObject([
-          {
-            article_id: 1,
-            title: "Living in the shadow of a great man",
-            topic: "mitch",
-            author: "butter_bridge",
-            body: "I find this existence challenging",
-            created_at: expect.any(String),
-            votes: 100,
-            article_img_url:
-              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
-          },
-        ]);
+        expect(Object.keys(body).length).toBeGreaterThan(0);
+        expect(body).toMatchObject({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: expect.any(String),
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+  test("GET:200 - Responds with the comment_count of a specific article when passed a query of comment_count", () => {
+    return request(app)
+      .get("/api/articles/1?comment_count=true")
```

Comment:
I think that has been a bit of confusion over this task. This shouldn't be a query (or new endpoint) - this should be included anyway now on the end point /api/articles/:article_id
You will just have a new test to check it's presence.